### PR TITLE
AF-2-dagger

### PR DESCRIPTION
**Pull request by MaksimGoleota**

### CODE REVIEW

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -1,13 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".feature.HomeFragment">
+    tools:context=".MainActivity">
```
Comment (YauheniHaliota) [inline app/src/main/res/layout/fragment_home.xml:7]: Revert this change

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -2,4 +2,6 @@
     <string name="app_name">Actors and Films</string>
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="textview1">TextView</string>
+    <string name="textview2">TextView</string>
```
Comment (YauheniHaliota) [inline app/src/main/res/values/strings.xml:6]: Remove both, and try to keep the code and resources clean

**Review by YauheniHaliota — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_home.xml (hunk 1 above)
Comment (MaksimGoleota) [reply app/src/main/res/layout/fragment_home.xml]: Done

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (MaksimGoleota) [reply app/src/main/res/values/strings.xml]: Done

**Review by YauheniHaliota — APPROVED** [review]
